### PR TITLE
fix: several updates

### PR DESCRIPTION
**Pull request by rollandf**
- fix some identation for list
- align style on bash - remove captions

### CODE REVIEW

File: docs/getting-started-kubernetes.rst [hunk 1]
```diff
@@ -133,15 +136,6 @@ Install the Network Operator from the NVIDIA NGC chart using custom values:
 
    helm show values nvidia/network-operator --version |network-operator-version| > values.yaml
 
-.. code-block:: bash
```
Comment (heyvister1) [inline docs/getting-started-kubernetes.rst:136]: Please return this part which had been removed

Comment (rollandf) [reply docs/getting-started-kubernetes.rst:136]: Reverted, and added a description before it

Comment (heyvister1) [reply docs/getting-started-kubernetes.rst:136]: Thanks

**Review by heyvister1 — APPROVED** [review]
LGTM

**Pull request merged**